conans/cli/commands/config.py

```diff
@@ -0,0 +1,111 @@
+import json
+
+from conans.cli.command import conan_command, COMMAND_GROUPS, conan_subcommand
+from conans.cli.output import ConanOutput
+from conans.model.conf import DEFAULT_CONFIGURATION
+from conans.util.config_parser import get_bool_from_text
+
+
+@conan_command(group=COMMAND_GROUPS['consumer'])
+def config(conan_api, parser, *args):
+    """
+    Computes a dependency graph, without  installing or building the binaries
+    """
+
+
+def json_build_order(build_order):
+    return json.dumps(build_order, indent=4)
+
+
+@conan_subcommand()
+def config_remote_install(conan_api, parser, subparser, *args):
```

Comment:
The names ``conan config remote-install``, ``conan config remote-reinstall``, etc, are to be defined, I couldn't think of a better name, but I know we want a name to differentiate them from ``conan config list`` (that lists [conf]).

Thought about sub-sub-commands, but seemed too much (requires adding infra from our cli code)

Reply:
If we go forward with this, as `remote-install` also manages local files, maybe it should be just `install` ?